application/src/main/scala/com/azavea/franklin/api/endpoints/TileEndpoints.scala

```diff
@@ -0,0 +1,74 @@
+package com.azavea.franklin.api.endpoints
+
+import com.azavea.franklin.error.NotFound
+import sttp.model.StatusCode.{NotFound => NF}
+import sttp.tapir._
+import sttp.tapir.codec.refined._
+import sttp.tapir.json.circe._
+
+import java.net.URLDecoder
+import java.nio.charset.StandardCharsets
+
+case class TileRequest(
```

Comment:
This worked out really well to collapse all the arguments for `EndpointInput` into a case class that better encapsulates the data without a bunch of `String` and `Int` flying around.